src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PostDetails } from '../PostData/PostDetails';
```

Comment:
It looks like the PostDetails component is used without passing any props to it. According to the task description, each post should be able to be opened in a sidebar with its comments. If the PostDetails component is expected to display details of the selected post and its comments, it probably needs the selected post data passed as a prop.